### PR TITLE
added dynamic reconfigure parameters for head_pose estimation

### PR DESCRIPTION
**Pull request by ngageorange**
model_points are scaled by a magic number * interpupillary distance.
Added ability to change the magic number using dynamic-configure....

Also added head_pitch dynamic parameter to alter what a "neutral head" means

### CODE REVIEW

File: rt_gene/cfg/ModelSize.cfg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```
Comment (Tobias-Fischer) [inline rt_gene/cfg/ModelSize.cfg]: Is that right? In a cfg file?

File: rt_gene/CMakeLists.txt [hunk 2]
```diff
@@ -13,13 +13,38 @@ find_package(catkin REQUIRED COMPONENTS
...
   uvc_camera
+  dynamic_reconfigure
 )
 
 ## System dependencies are found with CMake's conventions
 # find_package(Boost REQUIRED COMPONENTS system)
 
 catkin_python_setup()
 
+################################################
```
Comment (Tobias-Fischer) [inline rt_gene/CMakeLists.txt]: Can we get rid of those comments? Glad to see you followed the instructions though 👍

File: rt_gene/cfg/ModelSize.cfg [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+PACKAGE = "rt_gene"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("model_size", double_t, 0, "Scale of the 3D model of the face used for head pose estimation", 30.0, 1.0, 100.0)
```
Comment (Tobias-Fischer) [inline rt_gene/cfg/ModelSize.cfg]: Is that a reasonable range? That probably account from mice to elephant interpupillary distances currently doesn't it? :P

File: rt_gene/cfg/ModelSize.cfg (hunk 1 above)
Comment (ngageorange) [reply rt_gene/cfg/ModelSize.cfg]: Apparently so....

File: rt_gene/CMakeLists.txt (hunk 2 above)
Comment (ngageorange) [reply rt_gene/CMakeLists.txt]: Agreed

File: rt_gene/cfg/ModelSize.cfg (hunk 3 above)
Comment (ngageorange) [reply rt_gene/cfg/ModelSize.cfg]: Agreed too...

**Comment by Tobias-Fischer** [comment]
Thanks Ahmed!

**Pull request merged**